Makefile

```diff
@@ -258,6 +262,22 @@ ${PULP_SDK_INSTALL_DIR}: ${TOOLCHAIN_DIR}/pulp-sdk
 
 pulp-sdk: ${PULP_SDK_INSTALL_DIR}
 
+${TOOLCHAIN_DIR}/snitch_cluster:
+	cd ${TOOLCHAIN_DIR} && \
+	git clone https://github.com/pulp-platform/snitch_cluster.git && \
+	cd ${TOOLCHAIN_DIR}/snitch_cluster && git submodule update --init --recursive && \
+	git checkout ${SNITCH_COMMIT_HASH}
+
+${SNITCH_INSTALL_DIR}: ${TOOLCHAIN_DIR}/snitch_cluster
+	mkdir -p ${SNITCH_INSTALL_DIR}
+	cp -r ${TOOLCHAIN_DIR}/snitch_cluster/ ${SNITCH_INSTALL_DIR}/../
+	cd ${SNITCH_INSTALL_DIR} && \
+	source iis-setup.sh && \
```

Comment:
This shell script is specific for the iis server. Could you look into which parts of it are necessary for compiling the snitch runtime and add just those so not to pollute the environment :smirk: and cause issues in the future.

Reply:
you can check here https://github.com/pulp-platform/Deeploy/pull/14/commits/7f8fc3e54040c734db00cca54016cae144a1d220